scripts/tuxepedia/extractor.py

```diff
@@ -39,9 +39,11 @@ def get_logger(self):
         return logging.getLogger(self.__class__.__name__)
 
     def get_completed_monsters(self):
-        """Extract monster data from the Tuxepedia Wiki page
+        """
+        Extract monster data from the Tuxepedia Wiki page.
 
-        :return: dict/JSON of the Tuxepedia monster entries
+        Returns:
+            dict/JSON of the Tuxepedia monster entries
```

Comment:
@vnmabus whats your opinion on this?  should re treat the returns like everything else?  Capital letters and periods?

Reply:
I think so, yes: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html .